_docs_integrate/request-one-time-consent-of-peer.md

```diff
@@ -25,88 +25,107 @@ required_by:
+{: .notice--info}
+
+### Create the Request
+
+As already indicated, the Request contains a [ConsentRequestItem]({% link _docs_integrate/data-model-overview.md %}#consentrequestitem) within its `items` property.
+To create the Request, the Sender must proceed as described in the [Create outgoing Request]({% link _docs_use-cases/use-case-consumption-create-outgoing-request.md %}) use case, specifying a suitable payload as given in the example below.
+In this example, the value of the `mustBeAccepted` property of the ConsentRequestItem is set to `true`.
+Please note that the `<...>` notation is used as a placeholder for the actual data as usual and that the `link` property of the [ConsentRequestItem]({% link _docs_integrate/data-model-overview.md %}#consentrequestitem) is optional and can therefore also be omitted.
 
 ```jsonc
 {
   "content": {
-    // the content property of the payload in the step before
+    "items": [
+      {
+        "@type": "ConsentRequestItem",
+        "mustBeAccepted": true,
+        "consent": "<issue that the Sender wants the Recipient to agree to>",
+        "link": "<link to external website with more information on the issue>"
+      }
+    ]
   },
-  "peer": "<the address of the Recipient Connector>"
+  "peer": "<Address of Recipient>"
 }
 ```
 
-Note that the Request is currently in status `Draft`.
+Before creating the Request, the Sender should check whether it is valid. This can be done by proceeding as described in the documentation of the [Check if outgoing Request can be created]({% link _docs_use-cases/use-case-consumption-check-if-outgoing-request-can-be-created.md %}) use case. The advantage of checking the validity of the Request first before attempting to create it is that the Sender would receive a more precise [error]({% link _docs_integrate/error-codes.md %}) description in the case of a faulty Request.
+{: .notice--info}
 
-{% include copy-notice description="Save the complete `content` of the response. You will need it in the sending Request step." %}
+### Send the Request
 
-**Example response:**
+After the Request is created, the Sender can send it to the Recipient. To send the [Request via a Message]({% link _docs_integrate/requests-via-messages.md %}), the Sender have to follow the instructions of the [Send a Message to the Recipient]({% link _docs_use-cases/use-case-transport-send-message-to-recipients.md %}) use case documentation. To continue the example, the following payload must be used by the Sender to send the [created Request]({% link _docs_integrate/request-one-time-consent-of-peer.md %}#create-the-request) to the Recipient via a Message. It is essential that the `id` of the Request is specified, which was generated after the Request was created by the Sender with the [Create outgoing Request]({% link _docs_use-cases/use-case-consumption-create-outgoing-request.md %}) use case. This enables the Request to be processed correctly by the Recipient.
```

Comment:
After the Request **has been** created...

To send the Request via a Message, the Sender **has** to follow...